<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>0</y>
      <width>1141</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_SerialPort">
     <attribute name="title">
      <string>Serial Port Settings</string>
     </attribute>
     <widget class="QGroupBox" name="selectBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Select Serial Port</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="manufacturerLabel">
         <property name="text">
          <string>Manufacturer:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="serialPortInfoListBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="serialNumberLabel">
         <property name="text">
          <string>Serial number:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="locationLabel">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="vidLabel">
         <property name="text">
          <string>Vendor ID:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="pidLabel">
         <property name="text">
          <string>Product ID:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="parametersBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>206</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Select Parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="baudRateLabel">
         <property name="text">
          <string>BaudRate:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dataBitsLabel">
         <property name="text">
          <string>Data bits:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="dataBitsBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="parityLabel">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="parityBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="stopBitsLabel">
         <property name="text">
          <string>Stop bits:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="stopBitsBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="flowControlLabel">
         <property name="text">
          <string>Flow control:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="flowControlBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="baudRateBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_log">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>891</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>231</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Send Command</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>CMD</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ADDR</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>CNT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>VALUES</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_CMD">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_ADDR">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_CNT">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_VALUES">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PID</string>
     </attribute>
     <widget class="QCustomPlot" name="PIDPlot" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>871</width>
        <height>641</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PIDTest">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Axis Select</string>
      </property>
      <widget class="QRadioButton" name="radioButton_X_Axis">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>X Gyro</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Y_Axis">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Y Gyro</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Z_Axis">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Z Gyro</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_X_PID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>X PID</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Y_PID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Y PID</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Z_PID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Z PID</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
